.azure/OneBranch.Official.yml

```diff
@@ -24,6 +24,10 @@ parameters: # parameters are shown up in ADO UI in a build queue time
   displayName: 'Enable debug output'
   type: boolean
   default: false
+- name: 'WindowsContainerImage2DockerTag'
+  displayName: 'WindowsContainerImage2 DockerTag'
+  type: string
+  default: 'latest'
```

Comment:
Have you already pushed the image with the tag "latest" to the isolated ACR?

Reply:
I have pushed the `testing` image. The `latest` image is currently being built and pushed in `msquic-Official-Docker #20230113.4`.

Reply:
Cool, thanks, signing off. I will leave it up to decice whether check it in now or wait for `latest` to finish uploading.